core/src/transaction.rs

```diff
@@ -46,34 +45,49 @@ impl AsRef<[u8]> for CellKey {
 }
 
 #[derive(Clone, Serialize, Deserialize, Eq, PartialEq, Hash)]
-pub struct CellOutPoint {
+pub struct OutPoint {
     // Hash of Transaction
     pub tx_hash: H256,
     // Index of output
     pub index: u32,
 }
 
-impl fmt::Debug for CellOutPoint {
+impl fmt::Debug for OutPoint {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("CellOutPoint")
+        f.debug_struct("OutPoint")
             .field("tx_hash", &format_args!("{:#x}", self.tx_hash))
             .field("index", &self.index)
             .finish()
     }
 }
 
-impl Default for CellOutPoint {
+impl Default for OutPoint {
     fn default() -> Self {
-        CellOutPoint {
+        OutPoint {
             tx_hash: H256::zero(),
-            index: u32::max_value(),
+            index: 0,
```

Comment:
`Self::null()`

Reply:
`null` is for `cellbase`, default is for test.

Reply:
why change the index from max to 0?

Reply:
0 is a reasonable default value? And MAX is for `cellbase`